src/sentry/integrations/api/endpoints/organization_config_integrations.py

```diff
@@ -7,18 +10,49 @@
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases.organization import OrganizationEndpoint
 from sentry.api.serializers import serialize
-from sentry.integrations.api.serializers.models.integration import IntegrationProviderSerializer
+from sentry.apidocs.constants import RESPONSE_BAD_REQUEST
+from sentry.apidocs.examples.integration_examples import IntegrationExamples
+from sentry.apidocs.parameters import GlobalParams, IntegrationParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
+from sentry.integrations.api.serializers.models.integration import (
+    IntegrationProviderResponse,
+    IntegrationProviderSerializer,
+)
 from sentry.integrations.manager import default_manager as integrations
 
 
+class OrganizationConfigIntegrationsEndpointResponse(TypedDict):
+    providers: list[IntegrationProviderResponse]
```

Comment:
needed to this b/c our response is non standard. instead of just returning a list, we return a dict with 1 key "provider" which contains the list.